core/uvisor.h

```diff
@@ -68,7 +68,7 @@ typedef void (*UnprivilegedBoxEntry)(void);
 /* IOT-OS specific includes */
 #include <linker.h>
 
-/* system specific declatiions and IRQ vector table */
-#include <system.h>
+/* System IRQ vector table and default ISRs */
+#include "system.h"
```

Comment:
I should've read the comment more closely. The system.h header is no longer system-specific, so it makes sense to change this comment at the same time the system.h header is becoming no longer system-specific. I am OK with this change staying in.